### PR TITLE
fixes: daemonset.yaml and role-binding.yaml masked

### PR DESCRIPTION
**Pull request by rphillips**
* the .gitignore was masking the daemonset.yaml and role-bind.yaml
* removed the gitignore and regenerated the binary data
* removed the dependency on the `kubeconfig-in-cluster` configmap... It appears to have been removed from the installer
* Updates the namespace

### CODE REVIEW

**Comment by rphillips** [comment]
@sjenning this is ready for review... we will need to go through all the services that should be checkpointed and annotate them with `checkpointer.alpha.coreos.com/checkpoint`.

File: assets/pod-checkpointer-operator/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        k8s-app: pod-checkpointer
+      annotations:
+        checkpointer.alpha.coreos.com/checkpoint: "true"
+    spec:
+      containers:
+        - name: pod-checkpointer
+          command:
+            - /checkpoint
+            - --kubeconfig=/etc/kubernetes/kubeconfig
+            - --container-runtime-endpoint=unix:///var/run/crio/crio.sock
```
Comment (sjenning) [inline assets/pod-checkpointer-operator/daemonset.yaml:24]: does the checkpointer talk directly to the runtime?

File: assets/pod-checkpointer-operator/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+spec:
+  selector:
+    matchLabels:
+      k8s-app: pod-checkpointer
+  template:
+    metadata:
+      labels:
+        k8s-app: pod-checkpointer
+      annotations:
+        checkpointer.alpha.coreos.com/checkpoint: "true"
```
Comment (sjenning) [inline assets/pod-checkpointer-operator/daemonset.yaml:17]: does the checkpointer need to be checkpointed? my brain hurts thinking about the implications...

File: assets/pod-checkpointer-operator/daemonset.yaml (hunk 1 above)
Comment (rphillips) [reply assets/pod-checkpointer-operator/daemonset.yaml:24]: it does.

File: assets/pod-checkpointer-operator/daemonset.yaml (hunk 2 above)
Comment (rphillips) [reply assets/pod-checkpointer-operator/daemonset.yaml:17]: yep. the checkpointer gets checkpointed as well. It's so the static manifest exists on a reboot, and the kubelet can restart the checkpointer.

**Comment by rphillips** [comment]
updates the namespace.
Fixes #12

**Pull request merged**